.github/workflows/publish.yml

```diff
@@ -112,12 +112,12 @@ jobs:
         working-directory: src/D2L.Bmx/build
         run: |
           if ($env:PLATFORM -eq 'win') {
-            Compress-Archive -Path "bmx.exe" -DestinationPath "bmx-win.zip"
-            gh release upload "$env:RELEASE_TAG" "bmx-win.zip"
+            Compress-Archive -Path "bmx.exe" -DestinationPath "bmx-win-x64.zip"
```

Comment:
decided against x86-64 because the hyphen can be confusing.
also .NET uses x64 in RIDs.